### PR TITLE
Reduce charta artifact down to a  section

### PR DESCRIPTION
**Pull request by ludgerpaehler**

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -27,31 +27,32 @@ int main() {
...
 Enzyme is composed of four pieces:
 
-*   An optional preprocessing phase which performs minor transformations that tend to be helpful for AD.
-*   A new interprocedural type analysis that deduces the underlying types of memory locations
-*   An activity analaysis that determines what instructions or values can impact the derivative computation (common in existing AD systems).
-*   An optimization pass which creates any required derivative functions, replacing calls to `__enzyme_autodiff` with the generated functions.
+* An optional preprocessing phase which performs minor transformations that tend to be helpful for AD.
+* A new interprocedural type analysis that deduces the underlying types of memory locations
+* An activity analaysis that determines what instructions or values can impact the derivative computation (common in existing AD systems).
+* An optimization pass which creates any required derivative functions, replacing calls to `__enzyme_autodiff` with the generated functions.
```
Comment (wsmoses) [inline content/_index.md]: Maybe also add AD-specific and other novel optimizations?

File: content/_index.md [hunk 2]
```diff
@@ -27,31 +27,32 @@ int main() {
...
+* The Enzyme [getting started guide](/getting_started/)
+* The Enzyme [getting involved guide](/getting_involved/)
+* Previous [talks](/talks/).
+* You can try out Enzyme on our [Compiler Explorer instance](/explorer).
 
 ## Citing Enzyme
 
-To cite Enzyme, please cite the following two papers (first for Enzyme as a whole, then for GPU+optimizations):
-```
+To cite Enzyme, please cite the following two papers (first for Enzyme as a whole, second for GPU+optimizations, and third for AD of all other parallel programs (OpenMP, MPI, Julia Tasks, etc.)):
```
Comment (wsmoses) [inline content/_index.md]: two -> three papers
maybe `all other parallel programs` -> `arbitrary parallel programs`?

File: content/talks/_index.md [hunk 3]
```diff
@@ -8,28 +8,19 @@ weight: 100
 ## Papers
 
 * SC 21, [Reverse-Mode Automatic Differentiation and Optimization of GPU Kernels via Enzyme](https://dl.acm.org/doi/abs/10.1145/3458817.3476165); William S. Moses, Valentin Churavy, Ludger Paehler, Jan Hückelheim, Sri Hari Krishna Narayanan, Michel Schanen, and Johannes Doerfert
```
Comment (wsmoses) [inline content/talks/_index.md:10]: Can we update these?

at minimum we can pull the relevant ones from https://wsmoses.com/academic/ and enzyme.mit.edu/conference


We should also link to enzyme.mit.edu/conference

File: content/_index.md [hunk 4]
```diff
@@ -27,31 +27,32 @@ int main() {
 By differentiating code after optimization, Enzyme is able to create substantially faster derivatives than existing tools that differentiate programs before optimization.
 
 <div style="padding:2em">
-<img src="/all_top.png" width="500" align=center>
+    <img src="/all_top.png" width="500" align=center>
```
Comment (tgymnich) [inline content/_index.md]:
```suggestion
<img src="/all_top.png" width="500" align=center>
```

File: content/_index.md (hunk 2 above)
Comment (ludgerpaehler) [reply content/_index.md]: Addressed.

File: content/_index.md (hunk 1 above)
Comment (ludgerpaehler) [reply content/_index.md]: Addressed.

File: content/talks/_index.md (hunk 3 above)
Comment (ludgerpaehler) [reply content/talks/_index.md:10]: I think this is separate to this change, and belongs in a separate PR.

**Review by tgymnich — APPROVED** [review]

**Pull request merged**